src/data/behandlerdialog/usePaminnelseTilBehandler.tsx

```diff
@@ -36,3 +36,17 @@ export const usePaminnelseTilBehandler = (
     },
   });
 };
+
+export const useBehandlePaminnelseOppgave = () => {
```

Comment:
Laget en variant av `useBehandlePersonoppgave` for å kunne ha litt annen `onSuccess`-callback.